tracee-ebpf/Makefile

```diff
@@ -134,14 +135,13 @@ $(OUT_BPF_CORE): $(BPF_SRC) $(LIBBPF_HEADERS) $(CMD_CLANG)
 	$(CMD_CLANG) \
 		-I $(LIBBPF_HEADERS)/bpf \
 		-I $(BPF_HEADERS) \
-		-I tracee/$(ARCH) \
+		-I tracee/vmlinux \
+		-I tracee/vmlinux/$(ARCH) \
 		-DCORE \
 		-D__TARGET_ARCH_$(linux_arch) \
 		-D__BPF_TRACING__ \
 		-D__VM_LINUX_H__ \
```

Comment:
Do you now what is this macro used for?
Maybe we can remove it now?

Reply:
one dir contains core header that is common to all arches, the other contain just the arch header (which needs to be separated). is that what you mean ?

Reply:
No. I'm asking if you know what D__VM_LINUX_H__ is used for

Reply:
Oh, not sure it's origin, let's try to remove it.

Reply:
Checking git history, this define was added but never used or referred (in tracee or libbpf). I believe its safe to remove this. Removing and testing...